t/op/lop.t

```diff
@@ -109,6 +109,6 @@ for my $test (
 
 # precedence
 is((1 xor 1 and 0), 1, '(1 xor 1 and 0) == 1');
-is((1 xor 0 or 1), '', "(1 xor 0 or 1) == ''");
+is((1 xor 0 or 1), 1, "(1 xor 0 or 1) == 1");
 is((1 ^^ 1 && 0), 1, '(1 xor 1 and 0) == 1');
```

Comment:
The other tests have the stringy description matching what the test does. This one doesn't.

Suggestion:

```diff
- is((1 ^^ 1 && 0), 1, '(1 xor 1 and 0) == 1');
+ is((1 ^^ 1 && 0), 1, '(1 ^^ 1 && 0) == 1');
```

Reply:
fixed